systemtest/fleet_test.go

```diff
@@ -76,6 +76,14 @@ func TestFleetIntegrationMonitoring(t *testing.T) {
 		Output  map[string]interface{}
 	}
 	apmIntegration.getBeatsMonitoringStats(t, &metrics)
+	// We don't have a way to assert the writtenBytes at this layer
+	// use the value of `output.write.bytes`.
+	var writeBytes float64
+	if o := metrics.Libbeat["output"].(map[string]interface{}); len(o) > 0 {
+		if w := o["write"].(map[string]interface{}); len(w) > 0 {
+			writeBytes = w["bytes"].(float64)
+		}
+	}
```

Comment:
How about we extract that field, check that it's > 0, and then remove it from the map before the comparison below?